hardware-api.md

```diff
@@ -91,7 +97,23 @@ pin.analogRead(function(error, value) {
 
 ### PWM pins
 
-PWM pins are not yet implemented. See [#21](https://github.com/tessel/t2-firmware/issues/21).
+**PWM pins** are pulse-width modulated pins ([wiki link](http://en.wikipedia.org/wiki/Pulse-width_modulation)). Essentially, PWM is a digital signal that spends between 0% and 100% of its time on (this is its "duty cycle"). You can set the PWM pins to any value between 0 and 1 to approximate an analog signal. PWM is often used to control servo speeds or LED brightness.
+
+Tessel has four PWM pins: TX and RX (pins 5 and 6) on each module port.
```

Comment:
are they used as TX and RX in this application? It might be less confusing to just say pins 5 and 6 on each module port can be used for PWM.